### ISSUE TITLE
AXE chrome extension: aria-hidden attribute is incorrectly applied

### ISSUE THREAD

**Issue by khsbory** [dequelabs/axe-core#3947]
### Product

axe Extension

### Product Version

v4.51.0

### Latest Version

- [X] I have tested the issue with the latest version of the product

#### Expectation
Describe what you expected the product to do.
To test pro feature, I signed up with my google account.
And after sign up, because of inproper aria-hidden attribute, my screen reader cannot read any contents. I recorded demo video.
[Watch my demo video](https://nv-a11y.synology.me/wp-content/uploads/kboard_attached/5/202303/64183644f36643283218.mp4)
Even if I try to read the contents with my keyboard tab key or arrow key with my nvda browse mode, it is not possible.
#### Actual
Describe what the product actually does.
Screen reader does not read contents because of aria-hidden attribute.
#### How to Reproduce
Provide a code sample or link to a webpage that reproduces the issue. Without this, your issue may be closed without investigation.
Please sign up with gmail account. And then try to read the contents with screen reader.
#### Additional context
Any thing else we should know about the issue?

**Comment by straker** [comment]
Thanks for letting us know, and thanks for the video upload, very helpful. I've sent this ticket to team in charge of that page and they'll look into this.
